virtool/models.py

```diff
@@ -0,0 +1,22 @@
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import Column, String
+
+Base = declarative_base()
+
+
+class Label(Base):
+    __tablename__ = 'labels'
```

Comment:
I think this cannot be removed. It will raise an error:
`sqlalchemy.exc.InvalidRequestError: Class <class 'virtool.models.Label'> does not have a __table__ or __tablename__ specified and does not inherit from an existing table-mapped class.`